index.html

```diff
@@ -57,6 +57,260 @@ <h2>I'm a Software Developer</h2>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="pro-btn">
+            <span><a href="#">See project</a></span>
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="projects">
+          <div class="project-details">
+            <div class="project-title">
+              <h4>Profesional Art Printing Data</h4>
+            </div>
+            <div class="support-text">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. Has been the industry's standard.
+              </p>
+            </div>
+            <div class="pro-language">
+              <ul>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="pro-btn">
+            <span><a href="#">See project</a></span>
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="projects">
+          <div class="project-details">
+            <div class="project-title">
+              <h4>Profesional Art Printing Data</h4>
+            </div>
+            <div class="support-text">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. Has been the industry's standard.
+              </p>
+            </div>
+            <div class="pro-language">
+              <ul>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="pro-btn">
+            <span><a href="#">See project</a></span>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div>
+        <div class="about">
+          <div class="about-text">
+            <h4>About me</h4>
+          </div>
+          <div class="about-description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <div class="about-footer">
+            <div class="about-btn">
+              <span><a href="#">Get my resume</a></span>
+            </div>
+            <div class="about-border"></div>
+          </div>
+        </div>
+      </div>
+      <div>
+        <ul class="language-sec">
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec.svg" alt="rec" />
```

Comment:
- [ ] Kindly fix the [issue](https://github.com/Cavein254/mobile-first/pull/2/files#r1148426650) mentioned by the previous reviewer to match the given design. It is not solved yet.

- Your Design
- ![image](https://user-images.githubusercontent.com/60083437/227743618-1b33eb5f-6c77-44dd-a256-b97af794105d.png)

Reply:
@Tufahel Thanks for the review. On my end there are no such breaks and my code is running file. I have even deployed it on vercel and still I cannot find any breaks. 
https://mobile-first-petonz207-cavein254.vercel.app/

Reply:
![Screenshot_2023-03-26_02-00-52](https://user-images.githubusercontent.com/13707896/227746413-487e7e02-69e4-47f5-bf02-bbcd7d87e8c6.png)